apiclient.go

```diff
@@ -1,25 +1,23 @@
-package client
+package warrant
 
 import (
 	"bytes"
 	"encoding/json"
 	"fmt"
 	"io"
 	"net/http"
-
-	"github.com/warrant-dev/warrant-go/v4/config"
 )
 
 const (
-	ClientVersion string = "3.1.0"
+	ClientVersion string = "4.0.0"
```

Comment:
We should make sure we update this to the appropriate version before we release

Reply:
Can you also update the version in `go.mod`to `/v5` before release?